Code/LauncherUnified/Launcher.cpp

```diff
@@ -46,20 +46,25 @@ extern "C" void CreateStaticModules(AZStd::vector<AZ::Module*>& modulesOut);
 #   define REMOTE_ASSET_PROCESSOR
 #endif
 
+void CVar_OnViewportPosition(const AZ::Vector2& value);
+
 namespace
 {
-    void OnViewportResize(const AZ::Vector2& value);
+    void CVar_OnViewportResize(const AZ::Vector2& value);
```

Comment:
Looks like this is because the command line gets merged with the settings registry in ComponentApplication, which in turn gets applied before anything other than the application has been created (so no modules/Gems etc. exist yet). This makes sense because some systems probably need to parse CVars seeded from the command line while they're being created, but in this case we're trying to modify something that doesn't exist yet. Ideally we should probably move these native window related CVars out of the launcher entirely into NativeWindow.cpp, although we would then need to account for the fact that there can be multiple NativeWindow instances.